src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseApprovalServicePermissionTests.cs

```diff
@@ -42,52 +36,79 @@ public class ReleaseApprovalServicePermissionTests
         public async Task GetReleaseStatuses()
         {
             await PolicyCheckBuilder<SecurityPolicies>()
-                .SetupResourceCheckToFail(_release, CanViewReleaseStatusHistory)
-                .AssertForbidden(
-                    userService =>
+                .SetupResourceCheckToFailWithMatcher<Release>(r => r.Id == _release.Id, CanViewReleaseStatusHistory)
```

Comment:
Just wondering if there's any benefit to using this over `.SetupResourceCheckToFail(_release, CanViewReleaseStatusHistory)` given that we have `_release` available here?

Reply:
Because we're not using PersistanceHelper anymore, we're not getting back the same instance of Release that is _release anymore, but rather the instance that is pulled out of the contentDbContext in the method being checked.  Therefore we can't rely on the instance check anymore.

We could start throwing Equals methods around in our domain objects, but I'd personally prefer to do that if the production code needs it rather than helping the tests out.

Reply:
If I'm understanding you, that is?  :slightly_smiling_face:

Reply:
Yes that's what I meant thanks, I understand now.

I was making a similar change today and I think the reason why I didn't run into needing this change is that I saved `_release` outside this block just before `await PolicyCheckBuilder<SecurityPolicies>()`.

Reply:
:+1:  I think it'd be worth finding the best pattern to use here given we'll be doing a lot of this as we phase out PersistenceHelper